README.md

```diff
@@ -164,6 +178,11 @@ Model.reset_column_information
 * Use `box-sizing: border-box;` (Whenever browsers support it, this allows you to define the size of actual boxes, rather than the size before padding and borders are added. Generally works for IE8+).
 * Consider the organizational advantages of SMACSS. (We should link to our SMACSS guide here.)
 * Sometimes files should be named `my_styles.css.scss.erb`, sometimes it should be `my_styles.scss.erb` (Will - to add detail)
+* Write out the full hex code in uppercase. e.g. `#AAAAAA` instead of `#aaa`
+* Avoid meta-programming classes
+* Sass file extensions not include `.css` where possible. Prefer `foo.scss` to `foo.css.scss`.
+* Follow smacss guidelines for css organization (Base, Layout, Module, State, Theme). Avoid model-specific css. 
+* Always use a [css reset](https://github.com/markmcconachie/normalize-rails)
```

Comment:
* `Always use a css reset. If the css framework being used does not provide one, consider using normalize-rails (link).`